datasets/sentinel-3/collection/sentinel-3-olci-wfr-l2-netcdf/template.json

```diff
@@ -0,0 +1,950 @@
+            "href": "https://sentinel.esa.int/documents/247904/690755/Sentinel_Data_Legal_Notice",
+            "type": "application/pdf",
+            "title": "Sentinel Data License"
+        },
+        {
+            "rel": "about",
+            "href": "https://sentinel.esa.int/web/sentinel/user-guides/sentinel-3-olci/product-types/level-2-water",
+            "type": "text/html",
+            "title": "Level-2 Water WRR and WFR"
+        }
+    ],
+    "stac_extensions": [
+        "https://stac-extensions.github.io/sat/v1.0.0/schema.json",
+        "https://stac-extensions.github.io/table/v1.2.0/schema.json",
+        "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json"
+    ],
+    "keywords": [
+        "ESA",
+        "Copernicus",
+        "Sentinel",
+        "Water",
+        "Ocean"
+    ],
+    "msft:short_description": "Sentinel-3 Land Full Resolution water and atmospheric geophysical products.",
+    "msft:storage_account": "sentinel3euwest",
+    "msft:container": "sentinel-3",
+    "msft:region": "westeurope",
+    "providers": [
+        {
+            "name": "ESA",
+            "roles": [
+                "producer",
+                "processor",
+                "licensor"
+            ],
+            "url": "https://earth.esa.int/web/guest/home"
+        },
+        {
+            "name": "Microsoft",
+            "roles": [
+                "host"
+            ],
+            "url": "https://planetarycomputer.microsoft.com"
+        }
+    ],
+    "assets": {
+        "thumbnail": {
+            "title": "Sentinel-3 OLCI WFR L2 NetCDF Thumbnail",
+            "href": "https://ai4edatasetspublicassets.blob.core.windows.net/assets/pc_thumbnails/sentinel-3-olci-wfr-l2-netcdf-thumb.png",
+            "media_type": "image/png"
+        },
+        "geoparquet-items": {
+            "href": "abfs://items/sentinel-3-olci-wfr-l2-netcdf.parquet",
+            "title": "GeoParquet STAC Items",
+            "description": "Snapshot of the collection's STAC items exported to GeoParquet format.",
+            "type": "application/x-parquet",
+            "roles": [
+                "stac-items"
+            ],
+            "table:storage_options": {
+                "account_name": "pcstacitems"
+            },
+            "msft:partition_info": {
+                "is_partitioned": true,
+                "partition_frequency": "MS"
+            }
+        }
+    },
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    -180,
+                    -90,
+                    180,
+                    90
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "2016-04-25T00:00:00Z",
+                    null
+                ]
+            ]
+        }
+    },
+    "summaries": {
+        "sat:platform_international_designator": [
+            "2016-011A",
+            "0000-000A",
```

Comment:
I need to check on this. Perhaps the upstream source has some information about what's valid here.

Reply:
https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-3-olci/document-library/-/asset_publisher/hkf7sg9Ny1d5/content/sentinel-3-metadata-specification?_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_hkf7sg9Ny1d5_assetEntryId=3402206&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_hkf7sg9Ny1d5_redirect=https%3A%2F%2Fsentinels.copernicus.eu%2Fweb%2Fsentinel%2Fuser-guides%2Fsentinel-3-olci%2Fdocument-library%3Fp_p_id%3Dcom_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_hkf7sg9Ny1d5%26p_p_lifecycle%3D0%26p_p_state%3Dnormal%26p_p_mode%3Dview%26_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_hkf7sg9Ny1d5_assetEntryId%3D3402206%26_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_hkf7sg9Ny1d5_cur%3D0%26p_r_p_resetCur%3Dfalse links to an [excel file](https://sentinels.copernicus.eu/documents/247904/1964331/Sentinel-3-IPF-Metadata-Specification.xlsx/3a2b1021-9caf-4d06-a240-eba0d8bc233e?t=1611043901542) with some hints.  I see

- nssdcIdentifier value changed from SENTIENL 3 to 0000-000A
- Use the actual nssdcIdentifier value of 2016-011A standing for S3A instead of 0000-000A

So my unconfirmed guess is that ESA self-assigned an ID of 000-000A, and later were officially assigned 2016-011A. I'm comfortable updating `000-000{A,B}` to the right one.

Reply:
Fixed in 39517f4e9a90a27ff03f3f41fe18f6725e2b6dff.

Reply:
Thanks for tracking that down. I pasted your fix into the base class.

Reply:
2be4820ee87fd98b489934173f161fb9e5c209cf should have that in the base class.